src/app/static/src/app/components/adr/ADRInputDataset.vue

```diff
@@ -379,17 +286,10 @@
             },
             toggleModal() {
                 this.open = !this.open;
-                if (this.open) {
-                    this.preSelectDataset();
-                }
-            },
-            preSelectDataset() {
-                const selectedDatasetId = this.selectedDataset?.id
-                if (selectedDatasetId && this.datasets.some((dataset: any) => dataset.id === selectedDatasetId)) {
-                    this.newDatasetId = selectedDatasetId;
-                }
             },
-            confirmImport() {
+            confirmImport(datasetId: string, releaseId: string | null) {
```

Comment:
confirmImport method on ADRInputDataset component is expecting releaseId as a string as second param

Reply:
Oh thanks, good spot. Fixed that now